tests/fast-integration/kyma-environment-broker/helpers.js

```diff
@@ -23,7 +26,7 @@ async function provisionSKR(
   const shootName = resp.dashboard_url.split(".")[1];
   debug(`Operation ID ${operationID}`, `Shoot name ${shootName}`);
 
-  await ensureOperationSucceeded(keb, instanceID, operationID);
+  await ensureOperationSucceeded(keb, instanceID, operationID, 1000 * 60 * 30); //30m
```

Comment:
I'm afraid 30 min timeout for provisioning may be borderline. Currently, all pipelines are failing and provisioning SKRs is not available, but as far as I remember, on a good day, kyma 1.x could take 25 - 35 minutes to provision SKR, I hear kyma 2.x should be faster. I think KEB has 1h timeout for provisioning operation so maybe we make it KEB timeout + 5 minutes extra?

Reply:
note from @PK85, it would be great to have common place that defines timeouts for all steps and stages. Of course not as part of this ticket but if there is not such ticket defined, lets make one and put it to our backlog.

Reply:
@pbochynski are you sure about 30 minutes timeout for the provisioning in the test?

Reply:
a recent excerpt from fast integration tests that passed (rare occasion these days)
```
    ✓ Should provision new ServiceManager platform (4945ms)
    ✓ Should instantiate ServiceManager instance and binding for BTP operator (23319ms)
Operation ID 9e749ddf-a798-40a0-8d7c-f73a1c5be8f0 Shoot name c-6f50cf9
KEB operation: { state: 'succeeded', description: 'Operation created' }
Compass ID 62d8883a-645f-4b2a-b26e-08a418474110
    ✓ Should provision SKR (1711847ms)
    ✓ Should save kubeconfig for the SKR to ~/.kube/config
    ✓ Should initialize K8s client
```
`1711847ms` ~= 28 minutes, I remember seeing well above 2*10^6ms in our logs which is more than 30 minutes. But I don't think we should timeout the KEB call before it times it out and I support @PK85 idea that we define a common place for these timeouts eventually that the tests can reach (not in scope for this PR but good future work).